spec/support/vcr.rb

```diff
@@ -63,7 +63,7 @@
   config.around(:each, :vcr) do |example|
     if VCR.turned_on?
       cassette = cassette_name(example)
-      VCR.use_cassette(cassette, record: :new_episodes) do
+      VCR.use_cassette(cassette, record: :new_episodes, match_requests_on: %i[path query]) do
```

Comment:
My only question is whether you want to include `:method` in this too. As it stands it will use the same cassette for a GET or a POST to the same path and query. (Of course, a POST would not have a query but might have a body)

Reply:
i will give it a go and if it works fine with existing features then i will add it

Reply:
@jrmhaig that worked and done now